.github/workflows/publish.yml

```diff
@@ -0,0 +1,61 @@
+name: TDR Publish Slick Classes
+on:
+  workflow_dispatch:
+env:
+  BRANCH_NAME: version-bump-${{ github.run_id }}${{ github.run_attempt }}
+jobs:
+  deploy-db-changes:
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_PASSWORD: password
```

Comment:
Should the password and user values be hardcoded?

Reply:
They can be. It's just a local database that gets destroyed once the job finishes. We run `flywayMigrate` to populate the tables and stuff and then `slickCodegen` generates the classes from the database. The username and password are already defined [in build.sbt](https://github.com/nationalarchives/tdr-consignment-api-data/blob/master/build.sbt#L47) and we've got it hardcoded now in the [Jenkins node Dockerfile](https://github.com/nationalarchives/tdr-jenkins/blob/master/docker/nodes/postgres/Dockerfile#L10)

Reply:
Fair enough